pkg/exchange/okex/convert.go

```diff
@@ -259,15 +217,15 @@ func toGlobalOrderType(orderType okexapi.OrderType) (types.OrderType, error) {
 	switch orderType {
 	case okexapi.OrderTypeMarket:
 		return types.OrderTypeMarket, nil
-	case okexapi.OrderTypeLimit:
+
+	case okexapi.OrderTypeLimit, okexapi.OrderTypeMarketMakerProtection:
 		return types.OrderTypeLimit, nil
-	case okexapi.OrderTypePostOnly:
-		return types.OrderTypeLimitMaker, nil
 
-	case okexapi.OrderTypeFOK:
-	case okexapi.OrderTypeIOC:
+	case okexapi.OrderTypePostOnly, okexapi.OrderTypeIOC, okexapi.OrderTypeFOK, okexapi.OrderTypeMarektMakerProtectionPostOnly:
+		return types.OrderTypeLimitMaker, nil
```

Comment:
IOC or FOK is not an order type, it's a "TimeInForce" property, that means:

IOC and FOK could be used with Limit maker, limit taker, market taker order together

Reply:
You need to convert FOK and IOC into "TimeInForce"

Reply:
does okex's doc explain the IOC, FOK?

Reply:
The same as MAX, only limit order allowed in IOC and FOK, so I update IOC and FOK to `types.OrderTypeLimit` 
https://www.okx.com/cn/help-center/360025135731

Reply:
Got it, thanks. Could you add the doc explaination into the comment?